quinn/examples/https_client.rs

```diff
@@ -11,9 +11,10 @@ use structopt::StructOpt;
 use tracing::{error, info};
 use url::Url;
 
+use quinn::{Connection, Endpoint, NewConnection};
+
 mod common;
 
-/// HTTP/0.9 over QUIC client
```

Comment:
Doc comments on a structopt struct appear in the `--help` output, so this is meaningful and should not be removed.